discounts/rust/product-discounts/default/src/api.rs

```diff
@@ -51,23 +54,21 @@ pub struct Discount {
     pub conditions: Option<Vec<Condition>>,
 }
 
+#[skip_serializing_none]
+#[serde_as]
 #[derive(Clone, Debug, Serialize)]
 #[serde(rename_all(serialize = "camelCase"))]
 pub enum Value {
-    FixedAmount(FixedAmount),
-    Percentage(Percentage),
-}
-
-#[derive(Clone, Debug, Serialize)]
-#[serde(rename_all(serialize = "camelCase"))]
-pub struct FixedAmount {
-    pub applies_to_each_item: Option<Boolean>,
-    pub value: Float,
-}
-
-#[derive(Clone, Debug, Serialize)]
-pub struct Percentage {
-    pub value: Float,
+    #[serde(rename_all(serialize = "camelCase"))]
+    FixedAmount {
+        #[serde_as(as = "DisplayFromStr")]
+        amount: Decimal,
+        applies_to_each_item: Option<Boolean>,
```

Comment:
I figure that's what'll be generated from the schema if we ever do because that field is nullable 🤷🏽 Maybe at this point that doesn't matter?

Core will default to false, so those are the same